src/utils/draw.ts

```diff
@@ -93,15 +94,17 @@ export const get_svg_from_smarts = async (params: DrawSmartsSVGProps, worker: Wo
   if (!worker) return null;
   if (!params.smarts) return null;
 
-  const { canonicalForm: canonicalSmarts } = await getCanonicalFormForStructure(worker, {
-    structure: params.smarts,
+  const { structure } = await convertMolNotation(worker, {
+    moleculeString: params.smarts,
+    targetNotation: 'smarts',
     useQMol: true,
   });
```

Comment:
Why not use `getCanonicalFormForStructure(worker, {
    structure: params.smarts,
   molNotation: 'smarts',
   })` ? 
   
   
  on another note ^^ do we need to take `useQMol` as input in `getCanonicalFormForStructure`? can't we infer that from the `molNotation` before calling `convertMolNotation`?

Reply:
For the canonical form, we get it with `get_smiles`. Maybe we should change `getCanonicalFormForStructure` in rdkit-provider to only have 1 parameter `structure` and the expected return is a canonical SMILES.
Here we want to draw a smarts, so I replaced it by `convertMolNotation`  because of a bug in rdkit-js (https://github.com/rdkit/rdkit-js/issues/451)